### DISCUSSION TITLE
매크로 아무거나 선택과 특정백신 선택 속도차이

### DISCUSSION THREAD

**Discussion by soosooho** [changdoc/naver-vaccine-macro#105] [Q&A] [answered]
매크로 설정의 화이자, 모더나, 얀센 같은 선택창에서 특정 백신을 선택 후 매크로 돌릴때랑 아무거나 선택 후 매크로 돌릴때랑 예약과정에서의 속도차이가 있나요?

**Comment by changdoc** [comment, marked as answer]
백신 종류가 수십개로 늘어나면 모를까 속도 차이는 별도로 없을것으로 예상되네요.
